### ISSUE TITLE
New flag suggestion: rainbowcorner_au

### ISSUE THREAD

**Issue by maevelander** [devjack/pride.codes#36]
![rainbow_corner_au_right](https://user-images.githubusercontent.com/4695433/30627714-9ce23f38-9e04-11e7-89de-bead208791e2.png)

I'd like to suggest a flag as attached. For raising awareness and support without explicitly saying 'vote yes'. 

If it could be added to the CDN at https://cdn.pride.codes/js/rainbowcorner_au.js I'll create a PR over on the WordPress repo to add it there.

Thanks :-)

**Comment by devjack** [comment]
@EnigmaWeb Sorry we're a tad late to this - yes <3 it! 🏳️‍🌈 
@PatMurrayDEV is this something we could get forked from the current Vote Yes banner?

**Comment by maevelander** [comment]
No worries, let me know anything you need my end.

**Comment by PatMurrayDEV** [comment]
Yeah I think this is definitely something we can add, let me have a play around with it now
